tests/unit/utils/test_gmm.py

```diff
@@ -0,0 +1,56 @@
+import torch
+import numpy as np
+from nequip.utils import gmm
+from sklearn import mixture
+
+fit_data = np.random.rand(10, 3)
+gmm_sklearn = mixture.GaussianMixture(
+    n_components=3, covariance_type="full", random_state=123
+)
+
+gmm_torch = gmm.GaussianMixture(n_components=3, seed=123, n_features=3)
+# gmm_torch = torch.jit.script(gmm_torch)
```

Comment:
You can make a test that uses `assert_auto_jitable` to check whether your thing can be compiled correctly: https://github.com/mir-group/nequip/blob/fe73530ea34b16143de60cd9ed5e1dd14ce625b4/tests/unit/nn/test_embed.py#L19